spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,14 @@
+class UserMailerPreview < ActionMailer::Preview
+
+  def account_activation
+    user = User.first
```

Comment:
this will always trigger for the first user. I think it be better to send the user as a parameter

Reply:
It's just a preview tester fort the mailers, like in the book, this class sends no real mails. I tried to make a new user and send it as a parameter, but for some reason it is always nil, probably because you cannot create database models from here. What do you think is the best solution?

Reply:
These are the ones that get called if you go to localhost:3000/rails/mailers/...